src/main/resources/templates/templateOpportunityNotification.xml

```diff
@@ -7,10 +7,12 @@
                 <span class="tempo-text-color--normal">Opportunity:</span>
                 <#if (entity['OpportunityNotification'].current.opportunity.Link)??>
                     <a class="tempo-text-color--link" href="${entity['OpportunityNotification'].current.opportunity.Link}">${entity['OpportunityNotification'].current.opportunity.Name ! "-"}</a>
+                    <span>-</span>
                 <#else>
-                    <span>${entity['OpportunityNotification'].current.opportunity.Name ! "-"}</span>
+                    <span>${entity['OpportunityNotification'].current.opportunity.Name ! "-"} -</span>
```

Comment:
If there is no "Name" it will render "- -", right? Why don't you let this code like this:

`<span>${entity['OpportunityNotification'].current.opportunity.Name} -</span>`